docs/reply.rst

```diff
@@ -21,3 +21,13 @@ The ``reply`` Object
   .. js:function:: Reply.toJSON()
```

Comment:
let's take the oportunity to update the Reply object docs, we don't have a toJSON for example